src/host/polyfills/simpleView.ts

```diff
@@ -1,10 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import ToolsHost from "../toolsHost";
+
+declare var InspectorFrontendHost: ToolsHost;
+
+interface IRevealable {
+    lineNumber: number;
+    columnNumber: number;
+    uiSourceCode: {
+        _url: string;
+    };
+}
+
+export function revealInVSCode(revealable: IRevealable | undefined, omitFocus: boolean) {
+    if (revealable && revealable.uiSourceCode && revealable.uiSourceCode._url) {
+        InspectorFrontendHost.openInEditor(
+            revealable.uiSourceCode._url,
+            revealable.lineNumber,
+            revealable.columnNumber,
+        );
+    }
+
+    return Promise.resolve();
+}
+
 export function applyCommonRevealerPatch(content: string) {
     return content.replace(
         /Common\.Revealer\.reveal\s*=\s*function\(revealable,\s*omitFocus\)\s*{/g,
-        "Common.Revealer.reveal = function() { Promise.resolve(); return;");
+        `Common.Revealer.reveal = ${revealInVSCode.toString().slice(0, -1)}`);

```

Comment:
It's a hacky way of putting the revealInVSCode function into the devtools js. I'm tostring()-ing the function and stripping off the last end brace, so that I can prepend it into the script.

Patching the code like this in the post build step is not a great way to do it, but right now it does mean that you don't need a custom build of the Edge devtools to build/contribute to the extension. Ideally we would make such changes to the actual product code behind some sort of hosting flag (which the extension turns on by default). But for now this is the mechanism we're using.